CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 6.3.0
+
+New features:
+
+- [#430 Recommend Atom over Sublime text](https://github.com/alphagov/govuk_prototype_kit/pull/#430)
+
+- [#415  Update to govuk-elements-sass v3.1.1](https://github.com/alphagov/govuk_prototype_kit/pull/#415)
```

Comment:
The whitespace here needs stripping (tiny thing) and if we're going to keep the Greenkeeper emoji, we should use them for all Greenkeeper PRs.

Reply:
I fixed the whitespace, but I've left the PR messages as they were originally - they didn't have 'greenkeeper' (that's a label that got copied in error) but one did have emoji and the other one didnt